export const GET = async () => {
	throw new Error('TEST BACK END ERROR');
};
